### ISSUE TITLE
Add Weightstation Support for Mode2 Driver

### ISSUE THREAD

**Issue by Thomasrev** [Courseplay/courseplay#6324]
When I create a path for unloading from thresher to bunker and set a waiting point in the route the driver stops at the waiting point and does not leave anymore.

**Comment by Tensuko** [comment]
Why did you set a Waitpoint ?

**Comment by Thomasrev** [comment]
along the way I put a weighing station, to make it more real I would like to make the tractors above the scales wait three seconds

**Comment by Miketeg** [comment]
I created a wait point on a course to unload bale from the field to a sale point. I needed a wait point because I have to sale bale manually because of the layout.

I added to this course another course where AI was picking bale from the field.

Worker ignore the wait point in the course, even if wait time is set to 0 (not defined) or if I set a waiting time.

I'm using in game John Deere tractor + autostack 16 square bales from Arcusing (sorry, probably spelling it wrong)
